static/scss/style.css

```diff
@@ -1,1126 +1,897 @@
-  background: -moz-linear-gradient(-45deg, #8419c9 0%, #450e9e 100%);
-  background: -webkit-linear-gradient(-45deg, #8419c9 0%, #450e9e 100%);
-  background: linear-gradient(135deg, #8419c9 0%, #450e9e 100%);
-  filter: progid:DXImageTransform.Microsoft.gradient( startColorstr="#8419c9", endColorstr="#450e9e",GradientType=1 );
-}
-
-.footer ul {
-  padding: 0;
-}
-.footer ul li {
-  list-style: none;
-}
-.footer ul li a {
-  color: #93A5E4;
-  display: inline-block;
-  line-height: 2rem;
-  text-decoration: none;
-  font-weight: normal;
-  transition: all 0.5s;
-}
-.footer ul li a.footer-main {
-  color: #274BCA;
-  display: inline-block;
-  font-family: "Cerebri Sans", "Source Sans Pro", "Helvetica Neue", Arial, sans-serif;
-  margin-bottom: 1rem;
-}
-.footer ul li a:hover {
-  text-decoration: underline;
-  opacity: 0.9;
-}
-.footer .footer-social {
-  border-top: 1px solid #C7D1F1;
-}
-.footer .footer-social .social-link {
-  color: #BDC8EE;
-  font-size: 30px;
-  margin-right: 0.75rem;
-  text-decoration: none;
-  transition: all 0.5s;
-}
-.footer .footer-social .social-link:hover {
-  color: #adbaea;
-  text-decoration: none;
-}
-
-.engine-more .card {
-  min-height: 252px;
-}
-.engine-more .card .card-body {
-  flex: auto 0 0;
-}
-
-.licensing,
-.events {
-  background: #400d9a;
-  background: -moz-linear-gradient(top, #400d9a 0%, #5c12ad 100%);
-  background: -webkit-linear-gradient(top, #400d9a 0%, #5c12ad 100%);
-  background: linear-gradient(to bottom, #400d9a 0%, #5c12ad 100%);
-  filter: progid:DXImageTransform.Microsoft.gradient( startColorstr="#400d9a", endColorstr="#5c12ad",GradientType=0 );
-}
-.licensing::before,
-.events::before {
-  content: "";
-  display: block;
-  width: 100%;
-  height: 120px;
-  background: url(../assets/img/separator_5.png) center top repeat-x #fff;
-  background-size: 768px auto;
-}
-@media (min-width: 768px) {
-  .licensing::before,
-.events::before {
-    background-size: 1920px auto;
-    margin-top: -120px;
-    height: 300px;
-  }
-}
-
-section.projects {
-  background: url(../assets/img/separator_6.png) center top repeat-x transparent;
-  padding-top: 200px !important;
-}
-
-.network .ic {
-  color: #16bef5;
-  font-size: 48px;
-}
-
-.team .card {
-  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
-}
-.team .card img {
-  border-bottom: 1px solid #F4F7F9;
-}
-.team .card .ic {
-  font-size: 24px;
-  right: 20px;
-  top: 20px;
-}
-.team .card .card-intro {
-  font-size: 11px;
-  font-family: "Cerebri Sans", "Source Sans Pro", "Helvetica Neue", Arial, sans-serif;
-}
-.team .card .title-line:after {
-  margin-bottom: 1rem;
-}
-.team .card span.text-medium {
-  font-size: 13px;
-}
-
-.card.text-white {
-  border: none;
-}
-
-.google-calendar {
-  position: relative;
-  height: 0;
-  width: 100%;
-  padding-bottom: 50%;
-}
-
-.google-calendar iframe {
-  position: absolute;
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
-}
-
-/*# sourceMappingURL=style.css.map */
+@charset "UTF-8";

```

Comment:
It's strange that this file changed completely, usually it should be a small incremental change.
What is your sass version (`sass --version`)?

Reply:
I used Koala to generate the css from the style.scss file. I'm not sure of the version?